OperatingSystem-Services/Service-SOC/ServiceDocumentation/SuricataIntegrationWazuh.md

```diff
@@ -0,0 +1,136 @@
+# Integration of Suricata into Wazuh
+
+complain about this writeup to: viktor <br>
+
+## 1. Linux Steps (Ubuntu only...?)
+
+Install Suricata on the endpoint
+
+```bash
+sudo add-apt-repository ppa:oisf/suricata-stable
+sudo apt-get update
+sudo apt-get install suricata -y
+```
+
+Download and extract the Emerging Threats Suricata ruleset
+
+```bash
+sudo suricata-update
+```
+
+If fails try following
+```bash
+sudo suricata-update
+
+cd /tmp/ && curl -LO https://rules.emergingthreats.net/open/suricata-6.0.8/emerging.rules.tar.gz
+sudo tar -xvzf emerging.rules.tar.gz && sudo mv rules/*.rules /var/lib/suricata/rules/
+cd ~
+cd /var/lib/suricata/rules
+sudo chmod 640 *.rules
+```
+
+Check the network interface on the machine\
```

Comment:
Might want to put this in something like

[!IMPORTANT]
> 1. Check network interface
> 2. Check subnet range (convert to [CIDR](<Insert link here to CIDR tutorial>))

Just something like that. It's easy to skim past it.